src/host/libpixyusb_swig/pixy_viewer.py

```diff
@@ -36,48 +36,136 @@
 IMAGE_READ_DELAY_MS = 20
 
 
+## Enumeration of surrounding cells for use with color calculation from bayer pattern.
+class Neighbors:
+   C  = (1 << 0)  # center
+   N  = (1 << 1)  # top
+   NE = (1 << 2)  # top-right
+   E  = (1 << 3)  # right
+   SE = (1 << 4)  # bottom-right
+   S  = (1 << 5)  # bottom
+   SW = (1 << 6)  # bottom-left
+   W  = (1 << 7)  # left
+   NW = (1 << 8)  # top-left
+   NUM_CELLS = 9
+
+
+## Row and column direction for each neighbor from perspective of the center cell.
+## The index must align with the Neighbors enumeration class.
+NEIGHBORS_DIRECTION = [
+    (0, 0),  # center
+    (-1, 0),  # top
+    (-1, 1),  # top-right
+    (0, 1),  # right
+    (1, 1),  # bottom-right
+    (1, 0),  # bottom
+    (1, -1),  # bottom-left
+    (0, -1),  # left
+    (-1, -1),  # top-left
```

Comment:
Is this an (X,Y) coordinate system?

Reply:
It's  row then column or an image which would translate to (y, x)